packages/core/src/controllers/SIWEController.ts

```diff
@@ -0,0 +1,85 @@
+import { subscribeKey as subKey } from 'valtio/utils'
+import { proxy, ref, subscribe as sub } from 'valtio/vanilla'
+import type { SIWEClientMethods, SIWESession } from '../utils/TypeUtil.js'
+
+// -- Types --------------------------------------------- //
+// eslint-disable-next-line no-shadow
+export enum SIWEStatus {
```

Comment:
Don't use enums if possible, instead of this enum do following in `SIWEControllerClientState`

```
SIWEControllerClientState {
  status: 'uninitialized' | 'ready' | 'loading'
}
```

This way you will not need to export / import `SIWEStatus` anywhere and will still get autocomplete and type validation on status. If you absolutely need to use enums, I think they need to be set via `export const enum ...` to work properly when built for js.